backend/src/test/java/ch/puzzle/okr/AssertionHelper.java

```diff
@@ -0,0 +1,32 @@
+package ch.puzzle.okr;
```

Comment:
evtl. Klasse in package 'ch.puzzle.okr.test' verschieben?

ebenso CheckInTestHelpers, KeyResultTestHelpers, OverviewTestHelper, TestConstants, TestHelper
dann bleibt nur noch ForwardFilterTest auf gleicher Stufe wie die Impl

Reply:
Gute Idee. Umgesetzt.